### PR TITLE
[CFE-568]: Refactor(utils): home page overview, separate request

### PR DESCRIPTION
**Pull request by Poafs1**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Features
 
+- [#1021](https://github.com/alleslabs/celatone-frontend/pull/1021) Add Milkyway network (glados-1)
```
Comment (evilpeach) [inline CHANGELOG.md:42]: Please rebase from develop branch

**Review by songwongtp — CHANGES_REQUESTED** [review]

File: src/lib/services/stats/sequencer.ts [hunk 2]
```diff
@@ -8,13 +9,27 @@ export const getOverviewStatsSequencer = async (
...
       .then(({ data }) => data),
+    getBlocksSequencer(endpoint, undefined, 1),
     axios
       .get(`${endpoint}/indexer/block/v1/avg_blocktime`)
       .then(({ data }) => data),
-  ]).then((data) => parseWithError(zOverviewsStatsResponseSequencer, data));
+  ]).then(([dashboardApi, blocksLcd, avgBlockTimeLcd]) => {
+    const data = parseWithError(zOverviewsStatsResponseSequencer, [
+      dashboardApi.status === "fulfilled" ? dashboardApi.value : null,
+      avgBlockTimeLcd.status === "fulfilled" ? avgBlockTimeLcd.value : null,
```
[the remark this answers is not in the document]
Comment (evilpeach) [reply src/lib/services/stats/sequencer.ts:25]: Agree with it. It's no need to combine all together.

**Review by evilpeach — APPROVED** [review]

File: src/lib/services/types/stats.ts [hunk 3]
```diff
@@ -15,24 +15,19 @@ export const zOverviewsStatsResponse = z
...
+        tvl: z.number().nonnegative(),
+        tx_count: z.number().nonnegative(),
+        active_accounts: z.number().nonnegative(),
+      })
+    ),
+  })
+  .transform<OverviewsStats>((val) => ({
+    txCount: val.data[val.data.length - 1].tx_count,
+    latestBlock: val.last_block_height,
+    blockTime: 0,
```
Comment (songwongtp) [inline src/lib/services/types/stats.ts:32]: null

File: src/lib/pages/home/full.tsx [hunk 4]
```diff
@@ -86,21 +86,21 @@ export const HomeFull = () => {
           <CardInfo
             title={txInfo.title}
             tooltip={txInfo.tooltip}
-            value={overviewsStats?.txCount.toLocaleString()}
```
Comment (songwongtp) [inline src/lib/pages/home/full.tsx:89]: toString

File: src/lib/pages/home/sequencer.tsx [hunk 5]
```diff
@@ -86,22 +95,22 @@ export const HomeSequencer = () => {
           <CardInfo
             title={txInfo.title}
             tooltip={txInfo.tooltip}
-            value={overviewsStats?.txCount.toLocaleString()}
```
Comment (songwongtp) [inline src/lib/pages/home/sequencer.tsx:89]: toString()

**Review by songwongtp — APPROVED** [review]

**Pull request merged**